src/components/standalone/firewall/conntrack/ConntrackContent.vue

```diff
@@ -0,0 +1,189 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import { ubusCall } from '@/lib/standalone/ubus'
+import ConntrackRecordsTable from '@/components/standalone/firewall/conntrack/ConntrackRecordsTable.vue'
+import DeleteConntrackRecordModal from '@/components/standalone/firewall/conntrack/DeleteConntrackRecordModal.vue'
+import {
+  NeHeading,
+  NeInlineNotification,
+  NeButton,
+  NeSkeleton,
+  NeEmptyState,
+  NeTextInput
+} from '@nethesis/vue-components'
+import { getAxiosErrorMessage } from '@nethesis/vue-components'
+import { onMounted, ref, computed } from 'vue'
+
+const { t } = useI18n()
+
+export type ConntrackRecord = {
+  destination: string
+  destination_port?: string
+  destination_stats: {
+    bytes: string
+    packets: string
+  }
+  id: string
+  protocol: string
+  source: string
+  source_port?: string
+  source_stats: {
+    bytes: string
+    packets: string
+  }
+  state?: string
+  timeout: string
+}
+
+const isLoading = ref(false)
+const conntrackRecords = ref<ConntrackRecord[]>([])
+const error = ref({
+  notificationDescription: '',
+  notificationDetails: '',
+  notificationTitle: ''
+})
+const filter = ref('')
+const showDeleteModal = ref(false)
+
+onMounted(() => {
+  fetchConntrack()
+})
+
+const selectedItem = ref<ConntrackRecord>()
+
+async function fetchConntrack() {
+  error.value.notificationDescription = ''
+  error.value.notificationDetails = ''
+  try {
+    isLoading.value = true
+    conntrackRecords.value = (await ubusCall('ns.conntrack', 'list')).data.data
+  } catch (err: any) {
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+    error.value.notificationDetails = err.toString()
+  } finally {
+    isLoading.value = false
+  }
+}
+function applyFilterToConntrackRecords(records: ConntrackRecord[], filter: string) {
+  const lowerCaseFilter = filter.toLowerCase()
+  return records.filter(
+    (ConntrackRecord) =>
+      ConntrackRecord.source.toLowerCase().includes(lowerCaseFilter) ||
+      ConntrackRecord.destination.toLowerCase().includes(lowerCaseFilter) ||
+      ConntrackRecord.protocol.toLowerCase().includes(lowerCaseFilter) ||
+      (ConntrackRecord.state ?? '').toLowerCase().includes(lowerCaseFilter) ||
+      (ConntrackRecord.source_port ?? '').toLowerCase().includes(lowerCaseFilter) ||
+      (ConntrackRecord.destination_port ?? '').toLowerCase().includes(lowerCaseFilter)
+  )
+}
+
+function deleteAll() {
+  selectedItem.value = undefined
+  showDeleteModal.value = true
+}
+
+function openDeleteModal(itemToDelete: ConntrackRecord) {
+  selectedItem.value = itemToDelete
+  showDeleteModal.value = true
+}
+
+function closeDeleteModal() {
+  showDeleteModal.value = false
```

Comment:
^ here based on `selectedItem` value you can add the notification

Reply:
when we manually close or cancel the modal the notification will be displayed, not sure it the good place

Reply:
True, I meant under `onRecordDeleted`